### PR TITLE
Added support fo H2 for local development

### PR DESCRIPTION
**Pull request by yurichervoniy**

### CODE REVIEW

**Comment by yurichervoniy** [comment]
You can run application usinng command - ./mvnw clean spring-boot:run -Plocal

File: src/main/resources/application-local.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+spring:
```
Comment (arufanov) [inline src/main/resources/application-local.yml:1]: @yurichervoniy 
* how can I switch between applciaton.yml & application-local.yml?

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/main/resources/application-local.yml (hunk 1 above)
Comment (arufanov) [reply src/main/resources/application-local.yml:1]: @yurichervoniy 
* thanks, I understood.

**Review by arufanov — APPROVED** [review]
@yurichervoniy 
* thanks!

**Comment by arufanov** [comment]
@yurichervoniy 
* I accepted this pr to test

**Pull request merged**